src/api/notification/notification.ts

```diff
@@ -162,22 +157,22 @@ export class _Notification extends Base implements Notification {
                     message
                 }
             }
-        });
```

Comment:
Perhaps personal preference, but maybe make this an async function which awaits sendAction and doesn't return rather than .then(() => undefined)

Reply:
change it.